cluster/apiserver/apiserver.yaml

```diff
@@ -72,3 +72,10 @@ items:
           image: ghcr.io/vega-project/vega-project/ccb-operator/apiserver:latest
           args:
           - --dry-run=false
+          metadata:
```

Comment:
This is a leftover I assume?

Reply:
oh, I thought it was there already. Deleting it now.